.releaserc.json

```diff
@@ -0,0 +1,8 @@
+{
+  "tagFormat": "${version}",
```

Comment:
please remove this format, this will break the cycle.

the current version format is `vX.Y.Z`, if you remove the v it will start from the beginning.

Reply:
will do!